dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml

```diff
@@ -22,7 +22,7 @@
     the opportunity to learn computer science.
 
 %p
-  = link_to('Please take a minute to complete this short survey', @survey_url)
+  = link_to('Please take a minute to complete this short survey', @survey_url) + '.'
```

Comment:
At this point, that's how all the other links are so I'm keeping with the pattern. I initially picked it up from the [spec](https://docs.google.com/document/d/1blKBDZ167AoB01C4GcQ75yi0Jc4G2AW42imsenirvMk) which shows the period outside the link. I also personally prefer it that way (I think of the link as part of the sentence content, and the period is part of the container), but I don't feel too strongly about it.